test/unit/region.spec.js

```diff
@@ -122,18 +122,17 @@ describe('region', function() {
         onSwapOut: function() {}
       });
 
-      this.MyView = Backbone.View.extend({
+      this.MyView = Marionette.View.extend({
```

Comment:
Ah, is there coverage data somewhere?

I can work on reverting it. I guess I'd like to see where everyone stands on my other outstanding patch as if that is likely to get merged I'll just fix it there.